public/collections.json

```
@@ -64,5 +64,24 @@
       "en": "Online dating",
       "fr": "Rencontre en ligne"
     }
+  },
+  "PGA": {
+    "languages": ["en"],
+    "jurisdictions": ["EU"],
+    "maintainers": [
+      {
+        "name": "Platform Governance Archive",
```

I don't know if here should be 

```suggestion
        "name": "Universität Bremen",
```

or 

```suggestion
        "name": "Zentrum für Medien-, Kommunikations- & Informationsforschung, Universität Bremen",
```

…or something else entirely.

@ckatzenbach should tell us 🙂